main.js

```diff
@@ -0,0 +1,60 @@
+var keys;
+
+function growTree(object) {
+	keys = {
+	height: document.getElementById("height").value,
+	character: document.getElementById("character").value
+	};
+	var height = keys.height;
+	var char = keys.character;
+	var counter = keys.height;
+	var rowIndent = counter - 1;
```

Comment:
Since keys.height and keys.character are already representing the input values, it's not necessary to have multiple variables with the same value. RowIndent would just equal `keys.height - 1` and the code is easier to read because I already know what keys.height is.